topostats/summary_config.yaml

```diff
@@ -0,0 +1,39 @@
+output_dir: ./output/summary_distributions
+csv_file: ./all_statistics.csv
+file_ext: png
+pickle_plots: True
```

Comment:
Could we add some comments explaining what "pickle plots" means?

Reply:
There is a terse explanation in the table I've added to `docs/configuration.md` and I had noted this as a possible area to expand on under *ToDo (maybe)* of the initial post.

I've added the same terse description here for now and we can expand on this in a subsequent issue/work.

I was somewhat hesitant as I'm not sure many people would use this but felt it was a useful way for people to load and edit the pots generated.